R/precondition.R

```diff
@@ -132,16 +132,17 @@ require_directory_exists <- function(path, fmt = NULL) {
   path
 }
 
-ensure_directory <- function(path, ...) {
+ensure_directory <- function(path) {
```

Comment:
ensure_directory was only used in `spark_compile` now in `livy_install` as well.